servers/zms/src/main/java/com/yahoo/athenz/zms/config/MemberDueDays.java

```diff
@@ -19,9 +19,14 @@
 import com.yahoo.athenz.zms.Group;
 import com.yahoo.athenz.zms.Role;
 import com.yahoo.athenz.zms.utils.ZMSUtils;
+import com.yahoo.athenz.zms.ZMSConsts;
 
 public class MemberDueDays {
 
+    private static final int DEFAULT_USER_EXPIRY = Integer.parseInt(System.getProperty(ZMSConsts.ZMS_PROP_DEFAULT_USER_EXPIRY, "0"));
```

Comment:
same here. let's name these DEFAULT_MAX_USER_EXPIRY, etc to correctly indicate the max aspect of the value.